src/npc.cpp

```diff
@@ -688,7 +689,7 @@ int NpcScriptInterface::luagetDistanceTo(lua_State* L)
 	// getDistanceTo(uid)
 	ScriptEnvironment* env = tfs::lua::getScriptEnv();
 
-	Npc* npc = env->getNpc();
+	auto npc = env->getNpc();
```

Comment:
Could this be `const auto`
```suggestion
	const auto npc = env->getNpc();
```